Source/Private/Invoke-ExecCommandCore.ps1

```diff
@@ -0,0 +1,74 @@
+#   Borrowed from our friendly provider: https://github.com/dotnet/roslyn/blob/master/src/Setup/Installer/tools/utils.ps1
+
+function Invoke-ExecCommandCore {
+    param (
+        [Parameter(mandatory = $true)]
+        [string] $Command,
+        [string] $CommandArgs,
+        [switch] $PassThru = $true
```

Comment:
I don't think we should set PassThru to $true by default. The expected behavior is that the default value is $false. So, based on the principle of least surprise, I propose we leave as $false.

Reply:
As a docker cli user, the live log of a long running docker build is what i want to see during building of the image. Even when building with CI, it's still very useful to have it shown on the console when one is actually looking at the logs as it populates. 

We can change the switch to a negative one, like `-Quiet` so we don't see the logs printed on the host.

Tbh, i miss fast feedback when using the cmdlets.